functions-methods.js

```diff
@@ -19,8 +28,20 @@
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
+function typeOfEmail(emailAddress) {
+    const domain = getEmailDomain(emailAddress);
 
-
+    if (domain === "@novi-education.nl") {
+        userRole = "Student";
+    } else if (domain === "@novi.nl") {
+        userRole = "Medewerker";
+    } else {
+        userRole = "Extern"
+    }
+    return userRole;
```

Comment:
Als je niets meer met de waarde gaat doen na alle if-else's dan kun je de waarde ook meteen teruggeven:
```suggestion
    if (domain === "@novi-education.nl") {
        return "Student";
    } else if (domain === "@novi.nl") {
        return "Medewerker";
    }
    
    return "Extern";
```